ios/app/GoogleService-Info.plist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you look at the code which checks this, it basically checks if the file contains a valid key. I went this route, because you cannot simply drop a file in the src directory and expect Xcode to bundle it, it needs to be added to the project. So we add this "dummy" one and then in branding we'll override it at build time.

Reply:
Naive question: can it be empty then?

Reply:
I think it's easier if it's a valid Plist file, because that way we can check for the specific property we are interested in. I borrowed this pattern from Google's own Firebase examples and I like it.

Reply:
I think it's good to check if those values are not the dummy ones.